### ISSUE TITLE
color contrast rule fails on € sign

### ISSUE THREAD

**Issue by rrittwag** [dequelabs/axe-core#2100]
Color contrast rule fails on tags containing the € sign only, e.g.:
`<span><small>€</small>4.10</span>`

Expectation: Rule succeeds, as it does for currency signs $ and ¥.

**Comment by WilcoFiers** [comment]
More than likely, the euro sign also needs to have sufficient color contrast. But there are probably exceptions to this, so lets fix this and have it be consistent. Thanks for reporting!

**Comment by WilcoFiers** [comment]
Looked at this. The example (with a color applied to small) does not fail axe-core. Can you please provide a more complete snippet to reproduce the issue?

**Comment by padmavemulapati** [comment]
Verified in the latest axe-coconut(v4.4.1.08515) and now rule is Passing. Was able to reproduce with the older version, which means seeing the color-contrast issue for Euro symbol. 

![image](https://user-images.githubusercontent.com/47409579/77650484-92ad6680-6f91-11ea-913a-781e12f8ea98.png)

**Comment by straker** [comment]
No docs needed
